src/Form/inputTypeConfig.js

```diff
@@ -21,6 +21,13 @@ const getIdentitiesIcon = (input, {device}) => {
     return ''
 }
 
+/**
+ * Inputs with readOnly or disabled should never be decorated
+ * @param {HTMLInputElement} input
+ * @return {boolean}
+ */
+const canBeDecorated = (input) => !input.readOnly && !input.disabled
```

Comment:
This is the change I mentioned yesterday with the `readonly` attribute. We want to select them, but we don't want to decorate them (decorate here means adding icons and listeners).